tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -783,6 +787,84 @@ def test_org_mapping_already_taken(self):
         OrganizationMapping.objects.create(organization_id=999, slug="taken", region_name="us")
         self.get_error_response(self.organization.slug, slug="taken", status_code=409)
 
+    def test_configure_auth_provider(self):
+        with self.feature("organizations:api-auth-provider"):
```

Comment:
Can you use the decorator instead to reduce the code nesting

See example: https://github.com/getsentry/sentry/blob/d24190641b217df70d881580a5793a67899c6936/tests/sentry/tasks/test_digests.py#L43-L47